java/source/demo/pom.xml

```diff
@@ -21,7 +21,7 @@
     <properties>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
-        <arrow.version>7.0.0</arrow.version>
+        <arrow.version>8.0.0</arrow.version>
```

Comment:
We need to get this added to the list of post-release tasks and/or make it one of the things we check as part of release.